hoomd/md/pair/pair.py

```diff
@@ -452,6 +452,95 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0., mode='none'):
         # correctly?
```

Comment:
Calling out this note from previous code

Reply:
I think the answer should be no, as setting the max diameter properly is the job of the user in hoomd v3.